### PR TITLE
Forgot password feature

### PR DESCRIPTION
**Pull request by mariyasada**
adding forgot password page to E-commerce.
forget
login-after
login-before
adding cart page link to second card of product listing page.

### CODE REVIEW

File: HtmlFiles/Login.html [hunk 1]
```diff
@@ -114,8 +114,22 @@ <h2 class="heading heading-shadow">LOG IN</h2>
               <i class="fas fa-exclamation-triangle"></i> please enter valid
               password
             </p>
+            <span class="remeber-me-and-link-container text-size-sm">
```
Comment (tusharkandpal) [inline HtmlFiles/Login.html]: typo in `remember`

File: main.css [hunk 2]
```diff
@@ -7,6 +7,7 @@
 @import url(/stylesheets/product-detail/pDetail.css);
 @import url(/stylesheets/product-detail/textutil.css);
 @import url(/stylesheets/profile/profile.css);
+@import url(/stylesheets/forgotpass/pass.css);
```
Comment (tusharkandpal) [inline main.css]: these naming doesn't look good. can change!

File: stylesheets/logIN/login.css [hunk 3]
```diff
@@ -89,4 +89,17 @@ button{
     text-decoration: underline;
     color: var(--medium-teal);
 }
+.remeber-me-and-link-container
```
Comment (tusharkandpal) [inline stylesheets/logIN/login.css]: correct here too!

File: stylesheets/profile/profile.js [hunk 4]
```diff
@@ -8,17 +8,21 @@ let settingdetailDiv = detailcontainer[2];
...
 
 addressBtn.addEventListener( "click",() => {
     console.log("clicked");
     addressdetailsdiv.style.display==="none"? addressdetailsdiv.style.display = "block": addressdetailsdiv.style.display = "none";
-
+    profiledetailDiv.style.display="none";
+    settingdetailDiv.style.display="none";
 })
 settingBtn.addEventListener( "click",() => {
     // console.log("clicked");
```
Comment (tusharkandpal) [inline stylesheets/profile/profile.js]: remove this please!

**Review by tusharkandpal — CHANGES_REQUESTED** [review]
suggested few changes. then good to merge 💯

**Review by shaksham08 — APPROVED** [review]
Looks good to me

**Review by tusharkandpal — APPROVED** [review]

**Pull request merged**